discourse_rock/rockcraft.yaml

```diff
@@ -280,6 +279,108 @@ parts:
       chown -R 584792:584792 var/lib/pebble/default/.npm
       chown -R 584792:584792 var/lib/pebble/default/.yarn
       chown    584792:584792 var/lib/pebble/default/.yarnrc
+  imagemagick:
+    plugin: autotools
+    source: https://github.com/ImageMagick/ImageMagick.git
+    source-type: git
+    source-tag: 7.1.1-36
+    build-packages:
```

Comment:
This is confirmed to work, though I don't think we need the same mix of deps when building and when staging. I would expect a build to require the library headers of `libde265` (so `libde265-dev`) and the staged artefact to link to `libde265-0`. It's a minor thing, though you could look into it. A quick refactor would be to put the `-dev` packages in the build (and remove them from stage) and make sure the rock gets built just fine and Discourse is working (I expect it would). You don't need to do this as part of this PR..

Reply:
Working on it.

Reply:
I removed the `-dev` packages from `stage-packages` and it doesn't work unfortunately. I made sure they are still in `build-packages`.
I will work more on it tomorrow.